--- origsrc/Alien-wxWidgets-0.51/Build.PL	2010-03-30 15:32:33.000000000 -0500
+++ src/Alien-wxWidgets-0.51/Build.PL	2010-11-01 19:42:02.329765500 -0500
@@ -16,16 +16,6 @@ my %VERSIONS =
     );
 my $DEFAULT_VERSION = '2.8.10';
 
-if( $^O eq 'cygwin' ) {
-    print <<EOT;
-==========================================================
-There is no support to build Alien::wxWidgets (and wxPerl)
-on Cygwin.
-==========================================================
-EOT
-    exit 1;
-}
-
 # new_from_context is broken: it does not restore
 # @INC set in Build.PL before trying to load a base class not
 # defined using ->subclass...
