--- origsrc/Alien-wxWidgets-0.36/inc/My/Build/Any_wx_config.pm	2008-05-19 15:39:41.000000000 -0500
+++ src/Alien-wxWidgets-0.36/inc/My/Build/Any_wx_config.pm	2008-05-26 08:32:54.890625000 -0500
@@ -179,7 +179,13 @@
     return Alien::wxWidgets::Utility::awx_compiler_kind( $_[1] )
 }
 
-sub awx_dlext { $Config{dlext} }
+sub awx_dlext { 
+    if ($^O =~ m/cygwin/i) { 
+        "dll.a"
+    } else {
+        $Config{dlext}
+    }
+}
 
 sub _key {
     my $self = shift;
